en/guides/manage-devices/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Introduction
+table_of_contents: true
+---
+
+# Introduction
+
+On some systems it is appropriate to have a system user. For example, with a system user, one can ssh to the device and run commands.
+
+By default on first boot, stock Core images run a program called console-conf. If the user has a keyboard and monitor attached to the device, they can use console-conf to configure the network and add a user account. This user account can then be used to ssh to the device for various management purposes.
```

Comment:
I think console-conf needs a separate topic, so the detail that it works over serial is appropriate there.